index.js

```diff
@@ -105,8 +105,10 @@ function parse (args, opts) {
   var argv = { _: [] }
 
   Object.keys(flags.bools).forEach(function (key) {
-    setArg(key, !(key in defaults) ? false : defaults[key])
-    setDefaulted(key)
+    if (key in defaults) {
```

Comment:
This should use `Object.prototype.hasOwnProperty.call`, in case the "key" is "toString" or "hasOwnProperty"

Reply:
I kept `key in defaults` because if was there before, but I agree we should use `Object.prototype.hasOwnProperty.call`. I made the change.